recipe/build.sh

```diff
@@ -36,7 +36,7 @@ else
 fi
 
 # Build the rerun-cli and insert it into the python package
-cargo build --package rerun-cli $CROSS_TARGET --no-default-features --features native_viewer --release
+cargo build --package rerun-cli $CROSS_TARGET --no-default-features --features native_viewer,nasm --release
```

Comment:
We have a `release` feature-flag that includes `native_viewer` and `nasm`, and other future things we might want to include. I also includes `web_viewer`, which we probably want!

```suggestion
cargo build --package rerun-cli $CROSS_TARGET --no-default-features --features release --release
```

https://github.com/rerun-io/rerun/blob/1e529e3711768baa1da06dd57a0187136f5ed922/crates/top/rerun-cli/Cargo.toml#L39-L49

Reply:
Thanks, I saw `release` and I was wondering the same.